lib/vf_bitand.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * Cppcheck - A tool for static C/C++ code analysis
+ * Copyright (C) 2007-2024 Cppcheck team.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "vf_bitand.h"
+
+#include "mathlib.h"
+#include "token.h"
+#include "tokenlist.h"
+#include "vfvalue.h"
+
+#include "vf_settokenvalue.h"
+
+#include <string>
+
+namespace ValueFlow
+{
+    void analyzeBitAnd(TokenList &tokenlist, const Settings& settings)
```

Comment:
If we write `ValueFlow::analyzeBitAnd` we don't have to indent the whole function.

Reply:
In the initial discussion it was decided to sue the namespace instead.